test/e2e/scenarios/cmd_basic_create_update.yaml

```diff
@@ -0,0 +1,48 @@
+id: "RG_CMD_basic_create_update"
+description: "In this test we create RG with absolutely minimum configuration and try to update it."
+usecases:
+  - replication_group
+  - cmd
```

Comment:
I guess I still don't think it's necessary to have a separate "usecases" key instead of a generic "marks" key (like the one commented out below).

Reply:
Removed the `usecases`